### DISCUSSION TITLE
Grid inside Collapse causing animation to jitter

### DISCUSSION THREAD

**Discussion by mchandlernz** [mantinedev/mantine#1724] [Q&A]
Hi there. I'm trying to use a Collapsable section containing a Grid and noticed the Grid content will jitter in size when the collapsable section is opening. Any ideas on what's causing this / how to fix it?

**Demo code**
```
const grid = (
	<Grid>
		<Grid.Col>
			<Text>Test</Text>
		</Grid.Col>
	</Grid>
)

return (
	<Card>
		<Collapse>{grid}</Collapse>
	</Card>
)
```

**Comment by rtivital** [comment]
Please send a sandbox that reproduces the issue, template – https://codesandbox.io/s/mantine-template-9ze89?file=/src/App.tsx:0-6

**Reply by mchandlernz** [reply]
Hmm, it appears I cannot replicate it! It must be an issue on my end. Apologies
